helpers/manifest-parser.js

```diff
@@ -141,8 +141,13 @@ function parseIcon(raw) {
   let sizes = parseString(raw.sizes);
   if (sizes.value !== undefined) {
     let set = new Set();
-    sizes.value.split(/\s/).forEach(size => set.add(size.toLowerCase()));
-
+    sizes.value.split(/\s/).forEach(size => {
+      const pair = size.split(/x/i);
+      if (pair[0] !== pair[1]) {
```

Comment:
That sounds good for debugString.

My concern for where it lives is just mixing our layers here. Sam had mentioned making it so others could easily use our manifest parser, and having things that are browser-specific and aren't in the spec (like, things get weird if your icons aren't square) feel like they should live on code that builds on the parser, not in it.